matxmat44_neon.c

```diff
@@ -56,8 +55,19 @@ void matXmat_neon(float *A[], float B[N][N], float *D[]) {
   }
 }
 
-int main() {
+int main(int argc, char **argv) {
+  long int LOOPS = 100000000;
 
+  if (argc == 2) {
+    char *endptr;
+    LOOPS = strtol(argv[1], &endptr, 10);
```

Comment:
Ideally we will use something like getopt, eg:

https://www.man7.org/linux/man-pages/man3/getopt.3.html

But this will do for now.